PRS2001-3000/PRS2041ArmahA.xml

```diff
@@ -65,16 +66,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                <floruit notBefore="0600" notAfter="0699">He was a king of <placeName ref="LOC1310Aksum"/>
                   known from his coins, from several of the late Gǝʿǝz king lists, and from a single
-                  appearance in the manuscript colophon.
+                  appearance in a manuscript colophon.
```

Comment:
This writing is not a colophon but a record on construction of several churches (or just a marginal note); it has been published but is not easy to find, a reference can be included on Haile, G. 2017. The Marginal Notes in the Abba Gärima Gospels Aethiopica 19 (2017) 7–26. DOI:https://doi.org/10.15460/aethiopica.19.1.1126., pp. 14-15 (text, translation of the note)

Reply:
Thank you, added!